test/pages/__snapshots__/cart.test.js.snap

```diff
@@ -1156,11 +1156,11 @@ exports[`dispatch updateQuantity action on changing line item quantity 1`] = `
                   </link_Link>
                   <link_Link
                     className="o-button--sml c-cart-summary-buttons__cta c-cart-summary-buttons__cta--proceed"
-                    href="/checkout/shipping-address"
+                    href="/checkout/payment-method"
```

Comment:
I believe changes here and in the `shipping-method` snapshot are due to the changes in shift-react-components 0.5.0 which was not pulled into shift next yet.